src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Internal/Controllers/InternalConnectionsController.cs

```diff
@@ -171,4 +172,37 @@ async Task<ValidationProblemInstance> RemovePackage()
 
         return NoContent();
     }
+
+    /// <summary>
+    /// API for delegation check of access packages, for which packages the authenticated user has permission to assign to others on behalf of the specified party.
+    /// </summary>
+    [HttpGet("accesspackages/delegationcheck")]
+    [DbAudit(Claim = AltinnCoreClaimTypes.PartyUuid, System = AuditDefaults.EnduserApiStr)]
+    [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_ENDUSER_WRITE)]
+    [ProducesResponseType<PaginatedResult<AccessPackageDto.Check>>(StatusCodes.Status200OK)]
+    [ProducesResponseType<AltinnProblemDetails>(StatusCodes.Status400BadRequest, MediaTypeNames.Application.Json)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    public async Task<IActionResult> CheckPackage([FromQuery] Guid party, [FromQuery] IEnumerable<Guid>? packageIds, [FromQuery] IEnumerable<string>? packages, CancellationToken cancellationToken = default)
```

Comment:
Ikke egentlig behov for å duplisere DelegationCheck inn i internal da det ikke er noen funksjonell forskjell fra enduser endepunktet. Skaper bare dobbelt arbeid når vi evt. må endre på noe.